### DISCUSSION TITLE
Math render

### DISCUSSION THREAD

**Discussion by giovannipollo** [vriteio/vrite#60] [Ideas]
Hi! 
Thanks for the amazing tool. It would be nice to add math formula render, something like MathJax. Is this already planned?

Thanks!

**Comment by areknawo** [comment]
Hey,
Math formulas are in the backlog.
Are you looking for inline- or block-style math formulas specifically?

I was considering if I could add it as an extension but, given the complexity (especially with the handling of inline formulas) - that's unlikely for now.

That said, if there's interest, I might add them as a built-in feature in the next release.
